### PR TITLE
CHK-48-add-test-url-to-script

### PR DESCRIPTION
**Pull request by jnau**
only changes in script and cloud-config.yml

### CODE REVIEW

**Review by paul-sud — APPROVED** [review]
LGTM

File: script [hunk 1]
```diff
@@ -2,8 +2,9 @@
 while :
     do
         now=$(date +"%Y%m%d%H%M%S")
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkfiles.py "$@" --out $now-checkfiles.log --err $now-checkfiles-error.log --include-unexpired-upload 2> $now-file-errors.log 1> $now-file-output.log
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkexperiments.py "$@" --out $now-checkexperiments.log --err $now-checkexperiments-error.log 2> $now-exp-errors.log 1> $now-exp-output.log
-        /opt/encoded/checkfiles/bin/python /opt/encoded/checkfiles/checkmd5.py "$@" --out $now-checkmd5.log 2> $now-md5-errors.log 1> $now-md5-output.log
+        url=$(echo "$@" | rev | cut -d " " -f 1 | rev | sed -e 's|^[^/]*//||' -e 's|/.*$||' | cut -d "." -f 1)
```
Comment (paul-sud) [inline script]: This line is pretty beastly, especially the `sed`. Could you add a comment explaining what it does?

**Review by paul-sud — CHANGES_REQUESTED** [review]

File: script (hunk 1 above)
Comment (jnau) [reply script]: good idea. I did this awhile and forgot what this even did too. I ended up finding an unnecessary sed there. So I went ahead and revised this by making this url parsing into two lines with comments.

**Review by paul-sud — APPROVED** [review]

**Review by paul-sud — APPROVED** [review]
commit looks good

**Review by paul-sud — APPROVED** [review]
Awesome

**Pull request merged**